src/go/rpk/go.mod

```diff
@@ -48,15 +41,13 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.4.0
 	github.com/stretchr/testify v1.4.0
-	github.com/ugorji/go/codec v0.0.0-20181204163529-d75b2dcb6bc8 // indirect
 	golang.org/x/crypto v0.0.0-20200204104054-c9f3fb736b72
+	golang.org/x/net v0.0.0-20200226121028-0de0cce0169b // indirect
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 	golang.org/x/sys v0.0.0-20200217220822-9197077df867
 	google.golang.org/grpc v1.22.0 // indirect
 	gopkg.in/yaml.v2 v2.2.8
 	gotest.tools v2.2.0+incompatible // indirect
-	honnef.co/go/tools v0.0.1-2020.1.4
-	mvdan.cc/sh/v3 v3.1.1 // indirect
 )
 
-go 1.13
+go 1.15
```

Comment:
this may be too much, let me know if you want me to skip it for now.

Reply:
is there anything significant to go 1.15? i think we'd need to coordinate w/ internal tooling 

cc: @ivotron @0xdiba - do we need to update to 1.15 in vtools?

if there value, we can update. Any changelog worth noting here?

Reply:
tl;dr: That directive does not do much, in most cases (and in ours) go works with either version (but it is advised to set it to the minimum supported version). I'd say we leave the language changes to a later PR.

There are generally no notable changelogs but simple conventions could brake between versions.
Since we should be thinking of an update at some point I had checked a couple of weeks ago and only one test was failing :
```
--- FAIL: TestGrafanaHostNoServer (0.00s)
    grafana_test.go:33: 
        	Error Trace:	grafana_test.go:33
        	Error:      	"Get "http://localhost:8888/metrics": dial tcp [::1]:8888: connect: connection refused" does not contain "Get http://localhost:8888/metrics: dial tcp"
        	Test:       	TestGrafanaHostNoServer
```
which is due to a simple change in quoting policy (so the test needs changing).

That said, before pushing an upgrade we should test rpk for its functionality too, that's why I propose doing it on its own at a later time.